src/Microsoft.Win32.Primitives/src/Microsoft.Win32.Primitives.csproj

```diff
@@ -28,7 +32,7 @@
     </Compile>
     <Compile Include="System\ComponentModel\Win32Exception.Windows.cs" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' And '$(TargetGroup)' == 'netcoreapp' ">
+  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' And ( '$(TargetGroup)' == 'netcoreapp'  Or '$(TargetGroup)' == 'uap' ) ">
```

Comment:
> '$(TargetsUnix)' == 'true' And ( '$(TargetGroup)' == 'netcoreapp'  Or '$(TargetGroup)' == 'uap'  [](start = 24, length = 97)

this just temporary? I assume when you fix the configuration for UAP not associated with Unix you'll not need this.